test/db2.discover.test.js

```diff
@@ -196,6 +196,39 @@ describe('Discover model primary keys', function() {
           }
         });
     });
+    it('should return an array of primary keys for STRONGLOOP.CUSTOMER',
+      function(done) {
+        db.discoverPrimaryKeys('CUSTOMER',
+          {owner: config.schema},
+          function(err, models) {
+            if (err) {
+              console.error(err);
+              done(err);
+            } else {
+              models.forEach(function(m) {
+                // console.dir(m);
+                assert(m.tableName === 'CUSTOMER');
```

Comment:
The test is just checking for the tablename. Can we actually check for the primary keys?